reference/shared/document.getselecteddataasync.md

```diff
@@ -15,21 +15,46 @@ Office.context.document.getSelectedDataAsync(coercionType [, options], callback)
 
 ## Parameters
 
+| Name           | Type               | Description |
+|:---------------|:-------------------|:------------|
+| _coercionType_ | [CoercionType](#coerciontype)    | The type of data structure to return. Required.
+| _options_      | [object](#options) | Specifies any of the following [optional parameters](../../docs/develop/asynchronous-programming-in-office-add-ins.md#passing-optional-parameters-to-asynchronous-methods)
+| _callback_     | [function](#callback-value) | A function that is invoked when the callback returns, whose only parameter is of type  [AsyncResult][].
 
+[AsyncResult]: ./asyncresult.md
```

Comment:
Remove this extraneous line?

Reply:
@JuaneloJuanelo -- can you please review this PR (and Accept it, if the changes are accurate)? Thanks!

Reply:
Line 24 is providing the URL used by [AsyncResult][] in lines 22, 48, 55 and 57. I did however move it to the bottom of the page (along with the other link references) so they're easy to find/maintain.